artifacthub-pkg.yml

```diff
@@ -1,42 +1,38 @@
----
-version: 0.1.6
+# Kubewarden Artifacthub Package config
+#
+# Use this config to submit the policy to https://artifacthub.io.
+#
+# This config can be saved to its default location with:
+#   kwctl scaffold artifacthub > artifacthub-pkg.yml 
+version: 0.1.7
```

Comment:
@jvanz using `make VERSION=v0.1.6 annotated-policy.wasm` is incorrect for those policies whose version is not tracked via git tags.

(explaining mainly for @geeksambhu benefit, and maybe in too much detail)
On Go, Swift, Rego policies, given their language tooling, the canonical source for the version is a git tag.
Since we need `artifacthub-pkg.yml` to have the urls and version of the policy we are releasing, we need to preemptively recreate the `artifacthub-pkg.yml` with a yet unexistent version. Once we merge the PR, we tag it, and the github-actions we have written checks that tag and the `artifacthub-pkg.yml` match, and build and release the policy.

In Rust policies (like this one), the cannonical source for the version is not a git tag, but the [`version` field in `Cargo.toml`](https://github.com/kubewarden/allowed-proc-mount-types-psp-policy/blob/11e030bdd167c3a369210e39e87000be1b710d6c/Cargo.toml#L3).
Hence for Rust policies, we bump the version in `Cargo.toml`, rebuild with `make annotated-policy.wasm`, commit the changes to both `Cargo.toml`, `Cargo.lock`, and we are done.
The PR then gets merged to main. Once it is in main, we tag a new release (the tag matches the string in `version` field of  `Cargol.toml` ), and then the CI checks the version, tests, etc, and releases.

What we are adding here

Reply:
So, to summarize. This is a rust policy, we shouldn't use `make VERSION=v0.1.6 annotated-policy.wasm` but change the `version` on `Cargo.toml`, and do a `make annotated-policy.wasm`. This should update the `artifacthub-pkg.yml`. Commit all changes.

Reply:
Resolving conversation, as we talked on slack.